gfx.cpp

```diff
@@ -1139,8 +1175,9 @@ class GfxInternal
         if(!SUCCEEDED(factory->EnumAdapterByLuid(device->GetAdapterLuid(), IID_PPV_ARGS(&adapter_))))
             return GFX_SET_ERROR(kGfxResult_InternalError, "Unable to find interop adapter");
 
+        is_interop_ = true;
         device_ = device;
-        max_frames_in_flight_ = GFX_MAX(max_frames_in_flight, 1u);
+        max_frames_in_flight_ = GFX_MIN(GFX_MAX(max_frames_in_flight, 1u), (uint32_t)kGfxConstant_MaxBackBufferCount);
```

Comment:
This is incorrect, similarly to my previous message, we can't rely on that `kGfxConstant_MaxBackBufferCount` constant beyond being a default value during initialisation.